main.c

```diff
@@ -783,8 +783,6 @@ int main(int argc, char *argv[]) {
 	}
 
 	if (load_config(config_path) == -1) {
```

Comment:
We should probably hard-fail on config file error, and not error out if the config file doesn't exist.